api/proto/auth/config.go

```diff
@@ -0,0 +1,97 @@
+package auth
+
+import (
+	"context"
+	"crypto/tls"
+	"net"
+	"time"
+
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/trace"
+)
+
+// Config contains configuration of the client
+type Config struct {
+	// Addrs is a list of addresses to dial
```

Comment:
Not clear which addresses are being referred to here. Teleport auth servers, proxies, anything else?